mutating-webhook/README.md

```diff
@@ -0,0 +1,8 @@
+# k8s-mutating-webhook
+
+## Status
+In progress
+
+## Description
+Works kind of like helm, except at runtime it modifies and replaces Fn::ImportValue stuff from annotations of other Custom resource objects.
+This is similar to AWS CloudFormation, and is aiming to give operators better interopability. 
```

Comment:
Can we add a Steps section?

## Steps

1. Generate certs
    - make gen-certs

2. Deploy Mutating Webhook
    - make deploy

3. Install Operators
    - make install-operators

4. Deploy application
    - make cluster
    - make moodle